tls/s2n_connection.c

```diff
@@ -1377,3 +1384,94 @@ int s2n_connection_set_keyshare_by_name_for_testing(struct s2n_connection *conn,
 
     POSIX_BAIL(S2N_ERR_ECDHE_UNSUPPORTED_CURVE);
 }
+
+static int s2n_openssl_free(uint8_t** data)
+{
+    if (*data != NULL) {
+        OPENSSL_free(*data);
+        *data = NULL;
+    }
+    return S2N_SUCCESS;
+}
+
+static int s2n_sk_X509_pop_free(STACK_OF(X509) **cert_chain)
+{
+    if (*cert_chain != NULL) {
+        sk_X509_pop_free(*cert_chain, X509_free);
+        *cert_chain = NULL;
+    }
+    return S2N_SUCCESS;
+}
+
+static int s2n_cert_chain_wipe(struct s2n_cert_chain *cert_chain)
+{
```

Comment:
This method probably belongs in crypto/s2n_certificate, with s2n_cert_chain_and_key_free and s2n_cert_chain_and_key_new

Reply:
We don't use this method anywhere else other than here